examples/influxdb_data_model.ipynb

```diff
@@ -513,16 +511,17 @@
     "    fields_by_measurement = {}\n",
     "    for verify_measurement in data['measurements']:\n",
     "        # DM-18360 - SQuaSH API /measurements should return the verification package \n",
-    "        influxdb_measurement = verify_measurement['metric'].split('.')[0]\n",
+    "        influxdb_measurement, field_key = verify_measurement['metric'].split('.')\n",
```

Comment:
Do we need to worry about what happens if `verify_measurement['metric']` has more than one period?

Reply:
Right, this needs to be improved. I think the right answer the SQuaSH API to return the package name (DM-18360) and remove it from the full qualified metric name.